### ISSUE TITLE
Implement QA probability calculation

### ISSUE THREAD

**Issue by dimaischenko** [RobotVeraDS/vera-dialog#2]
Implement into NLU module LSTM QA probability calculation and the way in which model parameters are stored.

**Comment by dimaischenko** [comment]
implement pretrained model loading in the way like: https://github.com/pytorch/vision/blob/master/torchvision/models/densenet.py

**Comment by dimaischenko** [comment]
Add probabilities calculation (exp) from logits.
